src/procedural.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace procedural;
```

Comment:
Are you sure you want to do that? A TitleCased namespace implies use of classes, whereas namespaces are case-insensitive.